examples/passing/ExtendedInfixOperators.purs

```diff
@@ -0,0 +1,20 @@
+module Main where
+
+zipWith :: forall a b c. (a -> b -> c) -> [a] -> [b] -> [c]
+zipWith _ [] _  = []
+zipWith _ _  [] = []
+zipWith f (x:xs) (y:ys) = f x y : zipWith f xs ys
+
+test1 = [1, 2, 3] `zipWith (+)` [4, 5, 6]
```

Comment:
This is really nice and has great readability -- also easier to explain the `` rule as operating over expressions as opposed to just "identifiers". It would be useful to put a nested example in these tests to demonstrate the parser can handle nesting.